spec/lib/whitehall_importer/embed_body_references_spec.rb

```diff
@@ -10,6 +10,22 @@
       expect(govspeak_body).to eq("[Contact:#{content_id}]")
     end
 
+    it "ignores contact embed when there is no contacts" do
+      govspeak_body = described_class.call(
+        body: "[Contact:123]",
+        contacts: [],
+      )
+      expect(govspeak_body).to eq("")
```

Comment:
Instead of `eq("")` you could use `be_empty`, which looks nicer

Reply:
`be_empty` is certainly nicer!